### PR TITLE
fix: sidebar updates

### PR DESCRIPTION
**Pull request by bbehnke**
Explain what this PR does.

### CODE REVIEW

File: ui/src/components/common/SlidingSidebar/partials/K8sEvents/index.tsx [hunk 1]
```diff
@@ -195,14 +201,17 @@ export function NamespaceK8s({ namespaceId }: NamespaceK8sProps) {
         height: "100%",
       }}
     >
-      <span className="namespace-k8s-title">Namespace K8s Event Logs</span>
+      {!excludeHeader && (
+        <span className="namespace-k8s-title">K8s Event Logs</span>
```
Comment (mshakira) [inline ui/src/components/common/SlidingSidebar/partials/K8sEvents/index.tsx:205]: lets keep just K8s Events

File: ui/src/components/common/SlidingSidebar/index.tsx [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
-import { NamespaceK8s, NamespaceK8sProps } from "./partials/NamespaceK8s";
+import { K8sEvents, K8sEventsProps } from "./partials/K8sEvents";
 import { VertexDetails, VertexDetailsProps } from "./partials/VertexDetails";
+import { PiplineSpecs, PiplineSpecsProps } from "./partials/PipelineSpecs";
 import IconButton from "@mui/material/IconButton";
 import CloseIcon from "@mui/icons-material/Close";
 import slider from "../../../images/slider.png";
 
 import "./style.css";
+import { EdgeDetails, EdgeDetailsProps } from "./partials/EdgeDetails";
```
Comment (mshakira) [inline ui/src/components/common/SlidingSidebar/index.tsx:11]: order of import needs to be fixed

File: ui/src/components/common/SlidingSidebar/partials/K8sEvents/index.tsx (hunk 1 above)
Comment (bbehnke) [reply ui/src/components/common/SlidingSidebar/partials/K8sEvents/index.tsx]: updated

File: ui/src/components/common/SlidingSidebar/index.tsx (hunk 2 above)
Comment (bbehnke) [reply ui/src/components/common/SlidingSidebar/index.tsx]: updated

**Review by mshakira — APPROVED** [review]

**Pull request merged**